pom.xml

```diff
@@ -73,9 +95,113 @@
     </developers>
     <build>
         <plugins>
+            <plugin>
```

Comment:
I am not sure all that is needed. Probably worth adding plugin management only for the things this project actually uses.